config.json

```diff
@@ -926,6 +926,20 @@
       "unlocked_by": "kindergarten-garden",
       "uuid": "bae1f3a4-b63d-4efd-921a-133b3d5e379c"
     },
+
+    {
+      "core": false,
+      "difficulty": 5,
+      "slug": "yacht",
+      "topics": [
+        "games",
+        "parsing",
+        "sorting"
+      ],
+      "unlocked_by": "matrix",
```

Comment:
I've taken a quick look, and I think the `bob` exercise would be a better option than matrix. Could you change it?